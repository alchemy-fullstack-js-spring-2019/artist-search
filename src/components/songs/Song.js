import React from 'react';
import PropTypes from 'prop-types';

function Song({ song }) {
  return (
    <p>{song}</p>
  );
}

Song.propTypes = {
  song: PropTypes.string.isRequired
};

export default Song;
