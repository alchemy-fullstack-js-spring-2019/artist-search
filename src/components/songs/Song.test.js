import React from 'react';
import { shallow } from 'enzyme';
import Song from './Song';

describe('Song component',  ()=>{
  it('renders stuff', ()=>{
    const wrapper = shallow(<Song song='song' />);
    expect(wrapper).toMatchSnapshot();
  });
});
