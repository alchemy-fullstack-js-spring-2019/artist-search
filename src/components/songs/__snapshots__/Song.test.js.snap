// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Song component renders stuff 1`] = `
<p>
  song
</p>
`;
